### DISCUSSION TITLE
Metamask connection through Walletconnect/react-native-dapp module

### DISCUSSION THREAD

**Discussion by kryptify** [WalletConnect/walletconnect-monorepo#2457] [Developer Support] [answered]
hi everyone 
I have question for walletconnect
if I want to connect metamask with wallet connect in react native,
what I have to use??

can I still use  import {
  useWalletConnect,
} from '@walletconnect/react-native-dapp';??

I have implemented with this module but , under android 9 , it is not working but higher versions is working
please let me know solution

Thank you

**Comment by finessevanes** [comment, marked as answer]
It depends on what you're trying to do:
dapp dev: https://docs.walletconnect.com/2.0/reactnative/web3modal/dapp-usage
wallet dev: https://docs.walletconnect.com/2.0/reactnative/web3wallet/wallet-usage
